content/docs/reference/config/installation-options-changes/index.md

```diff
@@ -17,443 +17,225 @@ The tables below show changes made to the installation options used to customize
-| `global.oneNamespace` | `false` | `false` |  | `Whether to restrict the applications namespace the controller manages;If not set, controller watches all namespaces` |
-| `global.configValidation` | `true` | `true` |  | `Whether to perform server-side validation of configuration.` |
-
-### Modified `gateways` key/value pairs
-
-| Key | Old Default Value | New Default Value | Old Description | New Description |
-| --- | --- | --- | --- | --- |
-| `gateways.istio-ingressgateway.type` | `LoadBalancer #change to NodePort, ClusterIP or LoadBalancer if need be` | `LoadBalancer` |  | `change to NodePort, ClusterIP or LoadBalancer if need be` |
-| `gateways.istio-egressgateway.enabled` | `true` | `false` |  |  |
-| `gateways.istio-egressgateway.type` | `ClusterIP #change to NodePort or LoadBalancer if need be` | `ClusterIP` |  | `change to NodePort or LoadBalancer if need be` |
-
-### Modified `certmanager` key/value pairs
-
-| Key | Old Default Value | New Default Value | Old Description | New Description |
-| --- | --- | --- | --- | --- |
-| `certmanager.tag` | `v0.3.1` | `v0.6.2` |  |  |
-
-### Modified `kiali` key/value pairs
-
-| Key | Old Default Value | New Default Value | Old Description | New Description |
-| --- | --- | --- | --- | --- |
-| `kiali.tag` | `istio-release-1.0` | `v0.14` |  |  |
+| `global.hub` | `gcr.io/istio-release` | `docker.io/istio` | `Default hub for Istio images. Releases are published to docker hub under 'istio' project. Daily builds from prow are on gcr.io, and nightly builds from circle on docker.io/istionightly`  | `Default hub for Istio images. Releases are published to docker hub under 'istio' project. Daily builds from prow are on gcr.io, and nightly builds from circle on docker.io/istionightly` |
+| `global.tag` | `release-1.1-latest-daily` | `1.2.0-rc.2` | `Default tag for Istio images.`  | `Default tag for Istio images.` |
+| `global.proxy.resources.limits.memory` | `128Mi` | `1024Mi` |  |  |
+| `global.proxy.dnsRefreshRate` | `5s` | `300s` | `Configure the DNS refresh rate for Envoy cluster of type STRICT_DNS 5 seconds is the default refresh rate used by Envoy`  | `Configure the DNS refresh rate for Envoy cluster of type STRICT_DNS This must be given it terms of seconds. For example, 300s is valid but 5m is invalid.` |
 
-### Modified `security` key/value pairs
+### Modified `mixer` key/value pairs
 
 | Key | Old Default Value | New Default Value | Old Description | New Description |
 | --- | --- | --- | --- | --- |
-| `security.selfSigned` | `true # indicate if self-signed CA is used.` | `true` |  | `indicate if self-signed CA is used.` |
+| `mixer.adapters.useAdapterCRDs` | `true` | `false` | `Setting this to false sets the useAdapterCRDs mixer startup argument to false`  | `Setting this to false sets the useAdapterCRDs mixer startup argument to false` |
```

Comment:
@mandarjog @douglas-reid what is the impact of this value during upgrade?  I assume user needs to set this to true if mixer policy is used?

Reply:
you do not need to set this to `true` if mixer policy is used. this has been an option for at least one release now -- and we have publicly announced the deprecation policy around this shift. If anyone is still using old style Mixer config, they will either need to update this flag at install time or they will need to update their Mixer config to the new style. The old style represents a large burden on the API server which was causing issues in different install environments.

Reply:
@johnma14 @linsun The description does not state what the key/value pair does but rather the effect it has on a different key/value pair. Could we include both pieces of information?

Reply:
@rcaballeromx we would have to change this description in the values.yaml file for mixer. Not sure if we want to do this for 1.2. If we want, please let me know and I will make the changes.

Reply:
The details are highlighted in upgrade notice by @sdake